pandas/tests/test_strings.py

```diff
@@ -636,12 +636,200 @@ def check_index(index):
             check_index(index())
 
     def test_extract_single_series_name_is_preserved(self):
-        s = Series(['a3', 'b3', 'c2'], name='bob')
-        r = s.str.extract(r'(?P<sue>[a-z])')
-        e = Series(['a', 'b', 'c'], name='sue')
-        tm.assert_series_equal(r, e)
-        self.assertEqual(r.name, e.name)
-
+        s = Series(['a3', 'b3', 'c2'], name='series_name')
+        r = s.str.extract(r'(?P<letter>[a-z])')
+        e = DataFrame({"letter":['a', 'b', 'c']})
+        tm.assert_frame_equal(r, e)
+
```

Comment:
on these test add a comment with this issue number (and an explanation if the title is not enough)